Formula/schemathesis.rb

```diff
@@ -0,0 +1,198 @@
+class Schemathesis < Formula
+  include Language::Python::Virtualenv
+
+  desc "Testing tool for web applications with specs"
+  homepage "https://schemathesis.readthedocs.io/"
+  url "https://files.pythonhosted.org/packages/0e/81/f9198ed48e4370278f49889a6ed2f92effdbc9d0f1e538a62f1e55a80fde/schemathesis-3.16.1.tar.gz"
+  sha256 "e2480aa4aacf4053519170c1ccdfd1347ea30a303b7f76ce43c2155073c677e2"
+  license "MIT"
+
+  depends_on "flit" => :build
+  depends_on "hatch" => :build
+  depends_on "poetry" => :build
+  depends_on "python@3.10"
+  depends_on "six"
+
+  resource "attrs" do
+    url "https://files.pythonhosted.org/packages/d7/77/ebb15fc26d0f815839ecd897b919ed6d85c050feeb83e100e020df9153d2/attrs-21.4.0.tar.gz"
+    sha256 "626ba8234211db98e869df76230a137c4c40a12d72445c45d5f5b716f076e2fd"
+  end
+  resource "hypothesis" do
+    url "https://files.pythonhosted.org/packages/da/f6/53077bf0a5745fc45e32dc86882b945d16d26f994c884fe86d793f5e4d7f/hypothesis-6.49.1.tar.gz"
+    sha256 "0310d446fb87415e31798e8267288acc22f0be4a0530128a3a694190fcc43e5b"
+  end
+  resource "hypothesis_jsonschema" do
+    url "https://files.pythonhosted.org/packages/81/e0/c2802558d47e179e234f45c5d9dc39ff45d30d28692839a10be391d892fd/hypothesis-jsonschema-0.22.0.tar.gz"
+    sha256 "359504080a422924a77263789a00c4995b061991558ff6a8243949f94a4d74fc"
+  end
+  resource "hypothesis_graphql" do
+    url "https://files.pythonhosted.org/packages/cb/c5/35b7d9bff89b2369d350f75d5f79c7164e127c631b68d58ea9c3a88e567d/hypothesis-graphql-0.9.0.tar.gz"
+    sha256 "7f30e41450f979edbfd951b5161581baea318a04f7a9db56399ee464c313bfae"
+  end
+  resource "jsonschema" do
```

Comment:
Please follow the spacing for `resource`s used in other formulae. For example:
```suggestion
  resource "attrs" do
    url "https://files.pythonhosted.org/packages/d7/77/ebb15fc26d0f815839ecd897b919ed6d85c050feeb83e100e020df9153d2/attrs-21.4.0.tar.gz"
    sha256 "626ba8234211db98e869df76230a137c4c40a12d72445c45d5f5b716f076e2fd"
  end

  resource "hypothesis" do
    url "https://files.pythonhosted.org/packages/da/f6/53077bf0a5745fc45e32dc86882b945d16d26f994c884fe86d793f5e4d7f/hypothesis-6.49.1.tar.gz"
    sha256 "0310d446fb87415e31798e8267288acc22f0be4a0530128a3a694190fcc43e5b"
  end

  resource "hypothesis_jsonschema" do
    url "https://files.pythonhosted.org/packages/81/e0/c2802558d47e179e234f45c5d9dc39ff45d30d28692839a10be391d892fd/hypothesis-jsonschema-0.22.0.tar.gz"
    sha256 "359504080a422924a77263789a00c4995b061991558ff6a8243949f94a4d74fc"
  end

  resource "hypothesis_graphql" do
    url "https://files.pythonhosted.org/packages/cb/c5/35b7d9bff89b2369d350f75d5f79c7164e127c631b68d58ea9c3a88e567d/hypothesis-graphql-0.9.0.tar.gz"
    sha256 "7f30e41450f979edbfd951b5161581baea318a04f7a9db56399ee464c313bfae"
  end

  resource "jsonschema" do
```